packages/geolocator/tizen/src/setting.cc

```diff
@@ -9,6 +9,8 @@
 #include <package_manager.h>
 
 namespace {
```

Comment:
I changed to app_settings_manager.

Reply:
Well, the original `AppControl` class seems to make more sense than individual methods in new `AppSettingsManager` because it can better represent the lifecycle of an app_control handle. Just error handling was missing there. In that way you will have two classes:

- `AppSettingsManager` (has only public methods)
- `AppControl` (statically defined in `app_settings_manager.cc`, used by `AppSettingsManager`)

Reply:
I changed app_control_h to a local variable and deleted the private methods of AppSettingsManager.

Reply:
Alright. I'm okay with either way but you might discuss with the author of the original code (@bbrto21) about the change just in case.